components/ReadTime.vue

```diff
@@ -1,48 +1,59 @@
 <template>
   <div>
-    <textarea @input="getReadTime()" v-model="text"></textarea>
-    <h3 class="read-time">🎉 {{readTime}} 🎉</h3>
+    <div class="scrollbar-container">
+      <textarea @input="getReadTime()" v-model="text" class="scrollbar-hidden"></textarea>
+    </div>
+    <h3 class="read-time">🎉 {{ readTime }} 🎉</h3>
   </div>
 </template>
 
 <script>
-import getReadTime from "easy-read-time";
-import { STORY } from "~/static/story.js";
-export default {
-  data() {
-    return {
-      text: STORY,
-      readTime: "11 min read."
-    };
-  },
-  methods: {
-    getReadTime() {
-      this.readTime = getReadTime(this.text, 250).formattedString;
+  import getReadTime from "easy-read-time";
+  import { STORY } from "~/static/story.js";
+  export default {
+    data() {
+      return {
+        text: STORY,
+        readTime: "11 min read."
+      };
+    },
+    methods: {
+      getReadTime() {
+        this.readTime = getReadTime(this.text, 250).formattedString;
+      }
     }
-  }
-};
+  };
 </script>
 
 <style lang="scss" scoped>
-textarea {
-  background: #282c35;
-  border-radius: 4px;
-  border-radius: 4px;
-  border: 4px solid #969696;
-  color: white;
-  font-size: 1.3em;
-  height: 400px;
-  margin: 20px 0;
-  padding: 10px 15px;
-  resize: none;
-  width: 100%;
-  @media (max-width: 500px) {
-    height: 360px;
+  textarea {
+    background: #282c35;
+    color: white;
+    font-size: 1.3em;
+    padding-right: 30px;
+    outline: none;
+    resize: none;
+    width: 100%;
+    @media (max-width: 500px) {
+      height: 360px;
+    }
+  }
+
+  .scrollbar-container {
+    background: #282c35;
+    overflow: hidden;
+    border: 4px solid #969696;
+    border-radius: 4px;
+    margin: 20px 0;
   }
-}
-.read-time {
-  text-align: center;
-}
-</style>
 
+  .scrollbar-hidden {
+    height: 400px;
+    border: none;
+    margin: 0px -300px 0px 15px;
```

Comment:
Hey There,

According to this website https://www.textfixer.com/tutorials/browser-scrollbar-width.php the default width of a scrollbar is 17px, so please change `margin: 0px -300px 0px 15px => margin: 0px -300px 0px 17px`

Reply:
```suggestion
    margin: 0px -300px 0px 17px;
```

Reply:
![image](https://user-images.githubusercontent.com/20350641/62171877-d8b71980-b373-11e9-96b3-3e57f40e680c.png)

Reply:
Thanks, changed it!